libkbfs/root_metadata.go

```diff
@@ -377,12 +377,31 @@ func (md *RootMetadata) updateFromTlfHandle(newHandle *TlfHandle) error {
 // swapCachedBlockChanges swaps any cached block changes so that
 // future local accesses to this MD (from the cache) can directly
 // access the ops without needing to re-embed the block changes.
-func (md *RootMetadata) swapCachedBlockChanges() {
+func (md *RootMetadata) swapCachedBlockChanges(bps *blockPutState) {
```

Comment:
I always found this name confusing, since it seems to imply that you can call it multiple times. Perhaps `loadCachedBlockChanges`?